clj-refactor.el

```diff
@@ -2981,8 +3028,8 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-find-usages"
         (when (looking-at-p "(\\s-*def")
           (re-search-forward name)
           (paredit-backward))
-        (when (looking-at (format "\\(.+/\\)?\\(%s\\)" name))
-          (replace-match (format "\\1%s" new-name))))
+        (when (re-search-forward (format "\\(.+/\\)?\\(%s\\)" name) search-limit t)
```

Comment:
tested that as well, renaming foo works fine as well:

```clj
(defn testfoo [foobar foo]
  (println foobar foo))
```

hit me with some failing testcase if you still think there is a regression here pls.